handleData.js

```diff
@@ -151,10 +200,11 @@ Observer.prototype.greet = function() {
 
 // Making use of the stuff above
 var zach = new DungeonMaster("Zach");
-var molly = new Character("Molly", "Kalen", 3, "Rogue", "Human?");
+var molly = new Character("Molly", "Kalen", 3, "Rogue", "Elf");
 var nick = new Character("Nick", "Delmirev", 3, "Paladin", "Dragonborn");
 var nate = new Observer("Nate");
 var game = new Game(zach, [molly, nick]);
+
 game.establishAddressBooks();
 
 nick.sendMessage("Molly", "Hello, World!");
```

Comment:
I definitely want to investigate this further, but maybe a better place for it is on a PR for the feat/messages branch. I'll make some modifications to that branch, and submit a PR within the next few days.